src/cdrizzlebox.c

```diff
@@ -553,6 +659,15 @@ do_kernel_lanczos(struct driz_param_t *p) {
 
                 /* Allow for stretching because of scale change */
                 d = get_pixel(p->data, i, j) * scale2;
+                if (d2) {
+                    for (k = 0; k < ndata2; ++k) {
+                        if (p->data2[k]) {
+                            d2[k] = get_pixel(p->data2[k], i, j) * scale2;
```

Comment:
scale2 squared here too

Reply:
Yeah, I thought of this but then forgot to fix it (due to other things I needed to fix). Thanks for reminding